client/js/components/user/DpMasterToebList.vue

```diff
@@ -355,37 +404,20 @@ export default {
       this.updateFields()
     },
 
+    possiblyInsertDeletion (items) {
+      let returnItems = items
+      if (this.isEditable) {
+        returnItems = returnItems.map(item => ({ ...{ deletion: 'delete' }, ...item }))
+      }
+      return returnItems
```

Comment:
this can be simplified:
```suggestion
      if (this.isEditable) {
        return items.map(item => ({ ...{ deletion: 'delete' }, ...item }))
      }
```

Reply:
Thanks, done in [bd09859](https://github.com/demos-europe/demosplan-core/pull/2621/commits/bd09859369b585d08b71b93f350cb84aeb0534ba)